typescript/cli/src/commands/registry.ts

```diff
@@ -96,6 +112,34 @@ const addressesCommand: CommandModuleWithContext<{ name: string }> = {
   },
 };
 
+const rpcCommand: CommandModuleWithContext<{
+  name: string;
+  index: number;
+}> = {
+  command: 'rpc',
+  describe: 'Display the public rpc of a Hyperlane chain',
+  builder: {
+    name: {
+      type: 'string',
+      description: 'Chain to display addresses for',
+      alias: 'chain',
+      demandOption: true,
+    },
+    index: {
+      type: 'number',
+      description: 'Index of the rpc to display',
+      demandOption: false,
```

Comment:
nit: feel like arg specific defaults can go here
```suggestion
      demandOption: false,
      default: 0
```